### PR TITLE
Matrix blog post polishing and adding image handling plugin

### PR DESCRIPTION
**Pull request by tnabtaf**
## Blog polish ✨ 

Addressing feedback from @jaimergp.  Typo fixes and authorship cleanup basically.

**But**, we do need to decide how to to store images for things like blog posts.  Specifically, if an image is likely to only be used once, where should we store it?

This PR puts an image directly in the new `/static/img/blog/` directory.  PR #66 puts its images in a post-specific directory under `/static/img/`.  Which of these is a better option?  Or is another option better?

Docusaurus itself [says](https://docusaurus.io/docs/blog#blog-post-date) 

> 💡 TIP
> Using a folder can be convenient to co-locate blog post images alongside the Markdown file.

This assumes a model where each blog post has it's own folder under `/blog/` (we aren't using that model).

I'm not opposed to any of these, but we do need to decide.

## New image handling plugin 🔌 

This pull request also adds a new image handling plugin. The new blog post contains an example of how this is to be used.

### CODE REVIEW

File: blog/authors.yml [hunk 1]
```diff
@@ -1,10 +1,16 @@
-jaimergp:
-  name: Jaime Rodríguez-Guerra
-  title: Steering council member
-  url: https://github.com/jaimergp
-  image_url: https://github.com/jaimergp.png
+# Alphhabetical by author ID please.
```
Comment (jaimergp) [inline blog/authors.yml:1]:
```suggestion
# Alphabetical by author ID please.
```

**Comment by jaimergp** [comment]
We can decide on a standard as we go, as long as we keep the URLs untouched. I think it's possible to put things under a directory if the markdown file is called `index.md`.

E.g. `my-blog-post.md` should be the same as `my-blog-post/index.md`.

... but I haven't checked 😬

File: blog/authors.yml (hunk 1 above)
Comment (tnabtaf) [reply blog/authors.yml:1]: OK, you have to admit that misspelling alphabetical is kind of funny.

**Comment by tnabtaf** [comment]
@jaimergp The previous static site generator I used supported directories with index.md files in them.  I don't have a strong opinion on that.

And we don't need to resolve the "where to put images?" question today, but we do need to get a standard policy in place by the time we go public.

**Comment by travishathaway** [comment]
@jaimergp, luckily this is possible. I just checked 😎.

@Asmit2952, I prefer that we store images along with the blog posts themselves. This should make the organization easier to reason about. I've tried your approach in the past, and it just ends up being a huge pile of images in one folder, which is harder to reason about.

@tnabtaf, I'll go ahead and update this pull request so you can have a good example to follow in the future.

Additionally, I will add this information to the contributing guide and provide an example there for people to use.

I'd also like to add this plugin to our website:

https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-ideal-image

It looks like it helps deal with a lot of stuff related to image optimization that would be nice for us to have.

**Review by Asmit2952 — APPROVED** [review]
LGTM 🚀

**Pull request merged**